cmd/eno-reconciler/main.go

```diff
@@ -108,15 +109,15 @@ func run() error {
 	// This provides quick feedback in cases where only a few resources have changed.
 	writeBuffer := flowcontrol.NewResourceSliceWriteBufferForManager(mgr)
 
-	rCache := reconstitution.NewCache(mgr.GetClient())
 	recOpts.Manager = mgr
-	recOpts.Cache = rCache
 	recOpts.WriteBuffer = writeBuffer
 	recOpts.Downstream = remoteConfig
 	recOpts.Queue = workqueue.NewTypedRateLimitingQueue(
-		workqueue.DefaultTypedItemBasedRateLimiter[reconstitution.Request]())
+		workqueue.DefaultTypedItemBasedRateLimiter[resource.Request]())
+	recOpts.Cache = &resource.Cache{}
```

Comment:
Nit: Even if no options are being set now, maybe put it under a constructor? Just so it's harder to forget